### ISSUE TITLE
CSS reset on hyperlinks causes paragraph spacing issue (Outlook)

### ISSUE THREAD

**Issue by antonydjames** [foundation/foundation-emails#421]
**How can we reproduce this bug?**

Within a column, add two paragraphs - the first containing a link. E.g.

```
<columns>
<p>If you did not request this change please<a href="#">contact us</a>.</p>
<p>Best regards.</p>
</columns>
```

**What did you expect to happen?**

The paragraphs to have the normal, Foundation-defined margin between them.

**What happened instead?**

The first paragraph has no margin-bottom, so the paragraphs run against each other as if there was only a linebreak between them. This appears to be caused by a CSS reset within the Foundation framework which adds `Margin: 0; margin:0;` to all 'a' elements - this overrules the 'p' element's margin.

It's possible to remedy it by adding custom CSS but I'm not sure whether this is an adequate solution?

`p a { Margin: default; margin: default; }`

Here's a screenshot demonstrating what happens:

![image](https://cloud.githubusercontent.com/assets/9954370/15016430/f15b45c0-1208-11e6-9bb2-86612b3926be.png)

**What email clients does this happen in?**

Outlook 2007, 2010, 2013 and 2016.

**Comment by tdhartwick** [comment]
It looks like the syntax has nested paragraph tags. Can you verify that this syntax still has an issue? 

```
<columns>
<p>If you did not request this change please<a href="#">contact us</a>.</p>
<p>Best regards.</p>
</columns>
```

**Comment by antonydjames** [comment]
@tdhartwick Sorry, that's purely my error transcribing it here. Should have copy/pasted directly from the HTML. I've edited the original post to reflect the actual markup i.e. it is (and always has been) identical in the template to what's in your post.

**Comment by tdhartwick** [comment]
Ah, thanks! We'll take a look into this! Sounds like that your custom CSS fix might help, would you mind making a PR so we can quickly test it?

**Comment by antonydjames** [comment]
I'll certainly try - very new to Github so it may take me a moment to work out how!

**Comment by tdhartwick** [comment]
Oh awesome! Welcome to GitHub! If you're familiar with making branches it shouldn't be too different.

First you''ll need to make a new branch to work from with this command:
`git checkout -b [name_of_your_new_branch]` [name_of_your_new_branch] can be any name you want. It's best to be descriptive with the feature you're working on.
Make and commit the changes in that new branch and  run this command:
`git push origin [name_of_your_new_branch]`
Then you hop over to github and suggest that branch as a pull request. Checkout the documentation here: https://help.github.com/articles/creating-a-pull-request/

There's a lot of cool people around these parts that can help you out as well :)

**Comment by jsit** [comment]
@antonydjames code fixes this for me; added PR #762
